packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -59,17 +59,14 @@ const LEVELS_TO_CLASSES = {
   error: 'text-danger',
 }
 
-const UNLISTED_CHANNEL_VALUE = 'unlisted channel'
+const UNLISTED_CHANNEL_VALUE = ''
```

Comment:
why this const is empty ?

Reply:
To be the unlisted channel default value.

Reply:
So you choosed an empty string in order to define an `unlisted channel` ? I find this a little bit weird, but it's more like a convention, so if you are okey with this, I'm okey too :)

Reply:
Any strings might do, but the empty string is a good idea because no public channels will use it.